meson_options.txt

```diff
@@ -43,4 +43,12 @@ option('efi-libdir', type : 'string', description : 'path to the EFI lib directo
 option('efi-ldsdir', type : 'string', description : 'path to the EFI lds directory')
 option('efi-includedir', type : 'string', value : '/usr/include/efi', description : 'path to the EFI header directory')
 option('efi_os_dir', type: 'string', description : 'the name of OS directory in ESP')
+option('efi_sbat_spec_version', type : 'integer', value : 1, description : 'SBAT specification version')
+option('efi_sbat_fwupd_generation', type : 'integer', value : 1, description : 'SBAT fwupd generation')
+option('efi_sbat_distro_id', type : 'string', value : '', description : 'SBAT distribution ID, e.g. fedora')
+option('efi_sbat_distro_generation', type : 'integer', value : 1, description : 'SBAT distribution generation')
+option('efi_sbat_distro_summary', type : 'string', value : '', description : 'SBAT distribution summary, e.g. Fedora')
+option('efi_sbat_distro_pkgname', type : 'string', value : '', description : 'SBAT distribution package name, e.g. grub2')
+option('efi_sbat_distro_version', type : 'string', value : '', description : 'SBAT distribution version, e.g. 2.04-31.fc33')
+option('efi_sbat_distro_url', type : 'string', value : '', description : 'SBAT distribution URL, e.g. https://redhat.com/rpms/grub2')
```

Comment:
this should be updated for "fwupd" rather than grub2 example

Reply:
Indeed, thanks.